accern_xyme/accern_xyme.py

```diff
@@ -1367,6 +1367,15 @@ def s3_warnings(
                 warnings_io.flush()
         return res
 
+    def set_attr(
+            self, 
+            attr: str = True,
+            value: Any):
+        pipe_def = self.get_def()
+        pipe_def[attr] = value
+        self = self._client.set_pipeline(self.get_id(), pipe_def)
+        return None
```

Comment:
no you don't need to explicitly return None but you need to set the return type in the _definition_ to None